idci/keycloak-security-bundle/1.0/config/packages/idci_keycloak_security.yaml

```diff
@@ -0,0 +1,6 @@
+idci_keycloak_security:
+    server_url: 'http://localhost:8080/auth'
+    realm: 'realm-name'
+    client_id: 'my-client-id'
+    client_secret: 'my-client-secret'
```

Comment:
Let's use environment variables from some of these.

Reply:
Done, it was a good idea ! Thx for your feedback !